src/Browser/Navigators/Navigator/Assistant/Suggestion.js

```diff
@@ -60,76 +57,43 @@ export const isMatch =
 export const completion =
   (query:string, model:Model):Completion => {
     switch (model.type) {
-      case "Search":
+      case 'Search':
         return {
-          query
-        , match: Search.getMatch(query, model.search)
-        , hint: Search.getHint(query, model.search)
+          query,
+          match: Search.getMatch(query, model.search),
+          hint: Search.getHint(query, model.search)
         }
-      case "History":
+      case 'History':
       default:
         return {
-          query
-        , match: History.getMatch(query, model.history)
-        , hint: History.getHint(query, model.history)
+          query,
+          match: History.getMatch(query, model.history),
+          hint: History.getHint(query, model.history)
         }
     }
   }
 
-export const update =
```

Comment:
Can you please keep this function, even if it's not used right now. I would rather remove obsolete code in separate change from formatting one.